tests/integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -83,6 +83,7 @@ else
 		"k8s-shared-volume.bats" \
 		"k8s-volume.bats" \
 		"k8s-nginx-connectivity.bats" \
+		"k8s-hostname.bats" \
```

Comment:
nit: Can we ensure these two are inserted in alphabetical order?

Reply:
fixed